src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -3,103 +3,309 @@
+        - name: --os-type
+          type: string
+          short-summary: Install support for deploying ACIs of this operating system type.
+        - name: --service-principal
+          type: string
+          short-summary: Service principal used for authentication to Azure APIs. If not specified, a new service
+                         principal with contributor role is created and cached at {sp_cache} to be used by subsequent
+                         `az aks` commands.
+        - name: --client-secret
+          type: string
+          short-summary: Secret associated with a service principal. This argument is required if `--service-principal`
+                         is specified.
     examples:
-        - name: Install the ACI-Connector to an AKS Cluster.
-          text: az aks install-connector --name MyManagedCluster --resource-group MyResourceGroup --connector-name MyConnector
-        - name: Install the ACI-Connector for Windows to an AKS Cluster.
+        - name: Install the ACI Connector to a managed Kubernetes cluster.
+          text: >-
+            az aks install-connector --name MyManagedCluster --resource-group MyResourceGroup
+            --connector-name MyConnector
+        - name: Install the ACI Connector for Windows to a managed Kubernetes cluster.
           text: az aks install-connector --name MyManagedCluster --resource-group MyResourceGroup --connector-name MyConnector --os-type Windows
-        - name: Install the ACI-Connector for Windows and Linux to an AKS Cluster.
+        - name: Install the ACI Connector for Windows and Linux to a managed Kubernetes cluster.
           text: az aks install-connector --name MyManagedCluster --resource-group MyResourceGroup --connector-name MyConnector --os-type Both
-        - name: Install the ACI-Connector using specific SPN.
+        - name: Install the ACI Connector using a specific service principal and secret.
           text: az aks install-connector --name MyManagedCluster --resource-group MyResourceGroup --connector-name MyConnector --service-principal <SPN_ID> --client-secret <SPN_SECRET>
-        - name: Install the ACI-Connector from a specific custom chart.
+        - name: Install the ACI Connector from a custom Helm chart.
           text: az aks install-connector --name MyManagedCluster --resource-group MyResourceGroup --connector-name MyConnector --chart-url <CustomURL>
-"""
+""".format(sp_cache=SERVICE_PRINCIPAL_CACHE)
+
 helps['aks remove-connector'] = """
     type: command
-    short-summary: Undeploy the ACI-Connector from an AKS cluster.
+    short-summary: Remove the ACI Connector from a managed Kubernetes cluster.
+    long-summary: |-
+        Stops the cluster from deploying Azure Container Instances.
+        See https://github.com/Azure/aci-connector-k8s for more details.
+    parameters:
+        - name: --connector-name
+          type: string
+          short-summary: Name of the ACI Connector.
```

Comment:
The [global arguments](https://github.com/Azure/azure-cli/blob/KnackConversion/src/azure-cli-core/azure/cli/core/commands/parameters.py#L181-L199) that we inherit don't start with any article:
```
Arguments
    --resource-group -g [Required]: Name of resource group. You can configure the default group
                                    using `az configure --defaults group=<name>`.
    --location -l                 : Location. You can configure the default location using `az
                                    configure --defaults location=<location>`.
```
Those are repeated in the command authoring help, so I thought that was the recommended template.

I think it would be better to follow that guideline here and not prefix the arguments with "The" until that is added to the global arguments. If we add the definite article everywher as suggested the actual help looks like this:
```shell
$ az aks install-connector --help

Command
    az aks install-connector: Install the Azure Container Instances (ACI) Connector on a managed
    Kubernetes cluster.

Arguments
    --connector-name    [Required]: The name of the ACI Connector.
    --name -n           [Required]: The name of the managed cluster.
    --resource-group -g [Required]: Name of resource group. You can configure the default group
                                    using `az configure --defaults group=<name>`.
    --chart-url                   : URL of a Helm chart that installs ACI Connector.  Default:
                                    https://github.com/Azure/aci-
                                    connector-k8s/raw/master/charts/aci-connector.tgz.
    --client-secret               : The secret associated with the service principal. This argument
                                    is required if `--service-principal` is specified.
    --location -l                 : Location. You can configure the default location using `az
                                    configure --defaults location=<location>`.
...
```
That inconsistency bothers me more than the clipped use of English does.